google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/metrics/BuiltinMetricsTracer.java

```diff
@@ -175,6 +179,12 @@ public void attemptSucceeded() {
     recordAttemptCompletion(null);
   }
 
+  public void addTarget(String target) {
```

Comment:
setAttemptTarget - there should only be a single target per attempt ... add implies there are multiple targets